src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.Api/Altinn.App.Api.csproj

```diff
@@ -53,6 +53,7 @@
   <PropertyGroup>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <NoWarn>$(NoWarn);1591</NoWarn>
+    <Version>4.6.1</Version>
```

Comment:
This looks misplaced. Unnecessary.

Reply:
Was not the conclusion this is needed for vs and for nuget to get correct version @SandGrainOne

Reply:
`Version` is an alias for `PackageVersion`. (or vice versa) I prefer PackageVersion as the value is added to the name of the package. FileVersion and AssemblyVersion is used on the Library itself.

Reply:
Fix this for all projects

Reply:
Ok. Will try to remove it. It is VS that added this

Reply:
The issue was that I had created two PropertyGroup elements. One for library properties and a separate one for package properties. This caused issues with the library output. It was given default version of 1.0.0. The package was still named correctly. 

The two PropertyGroups I made has now been merged back together, so I hope it works again.